packages/cfd/src/sass/cfd-dashboard.scss

```diff
@@ -573,11 +573,64 @@
             border-radius: 0.5rem;
         }
     }
+
+    &__card-content-over-header {
+        display: flex;
+        background: var(--text-info-blue-bg);
+        color: var(--text-info-blue);
+        height: 4rem;
+        width: 100%;
+        justify-content: center;
+        font-weight: bold;
+        align-items: center;
+        border-radius: 1.6rem 1.6rem 0rem 0rem;
+    }
+
+    &__card-content-over-header-blank {
+        height: 4rem;
+        width: 100%;
+        border-radius: 1.6rem;
+        @include mobile {
+            height: 2rem;
+        }
+    }
+
     &__h2-header {
-        margin: 1.4rem 0 2.4rem;
+        text-align: center;
+        min-height: 8rem;
+        margin: 0rem 2rem;
+        @include mobile {
+            margin-bottom: 2rem;
+            min-height: auto;
+        }
+    }
+
+    &__card-flipped-container {
+        gap: 2rem;
+        padding: 4rem 2rem;
     }
+
+    &__card-content-container {
+        display: flex;
+        flex-direction: column;
+        height: 100%;
+    }
+
+    &__card-section-container {
+        display: flex;
+        flex-direction: column;
+        gap: 1.2rem;
+        height: 100%;
+        margin: 0rem 2rem 2rem 2rem;
+    }
+
     &__footer-wrapper {
+        min-height: 18.6rem;
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
         @include mobile {
+            min-height: 0;
```

Comment:
why do we need this?

Reply:
@amina-deriv a min-heigh is set for a container in desktop mode which is being displayed in mobile when no card is selected. since this is not required in mobile, i set it to '0'. shall i set it to 'auto' instead?